docs/UserGuide.md

```diff
@@ -515,10 +516,24 @@ Format: `alias TARGET ALIAS`
 
 Example:
 
-* `alias switch st` ---  This will add `st` as a valid shortcut for command
-  `switch`.
-* `alias addj j` --- This will add `j` as a valid shortcut for command `addj`
+1. Assuming you want to add `st` as an alias for `switch`, type `alias switch st` 
```

Comment:
I think you can omit the first part of the sentence here.

~~Assuming you want to add `st` as an alias for `switch`, t~~**T**ype `alias switch st` into the command box and press enter.